pyproject.toml

```diff
@@ -18,6 +18,9 @@ reportlab = "^3.6.11"
 toml = "^0.10.2"
 celery = {extras = ["redis"], version = "^5.2.7"}
 redis = "^4.3.4"
+haversine = "^2.7.0"
```

Comment:
We actually have this formula implemented in the `Bbox` class for `bbox.get_area()`, so I would avoid this additional dependency